crates/plonk-verifier-codegen-bin/Cargo.toml

```diff
@@ -1,14 +1,17 @@
 [package]
-name = "codegen-bin"
+name = "plonk-verifier-codegen-bin"
 version.workspace = true
 edition = "2018"
 license = "MIT OR Apache-2.0"
-publish = false
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-zksync_solidity_vk_codegen.workspace = true
+plonk_verifier_codegen.workspace = true
+
 structopt = "0.3"
 dialoguer = "0.8"
 
+[[bin]]
+name = "generate"
+path = "src/main.rs"
```

Comment:
Could you roll it back as well, please? Since this crate now exists within the workspace, having a custom command is confusing -- naming the binary is more intuitive.